### PR TITLE
Check for the right two tag indexes

### LINKED ISSUE #207: Incorrect index check in read side
The read side currently checks for an index called `tag-seq-nrs-meta` but there is no such index mentioned anywhere, did we rename it?

Ping @chbatey

### PR DESCRIPTION
**Pull request by johanandren**
Fixes #207

### CODE REVIEW

File: core/src/main/scala/akka/persistence/couchbase/scaladsl/CouchbaseReadJournal.scala [hunk 1]
```diff
@@ -146,19 +146,23 @@ final class CouchbaseReadJournal(eas: ExtendedActorSystem, config: Config, confi
...
    * stored events is provided by `currentEventsByPersistenceId`.
    */
-  override def eventsByPersistenceId(persistenceId: String,
-                                     fromSequenceNr: Long,
-                                     toSequenceNr: Long): Source[EventEnvelope, NotUsed] =
+  override def eventsByPersistenceId(
+      persistenceId: String,
+      fromSequenceNr: Long,
+      toSequenceNr: Long
+  ): Source[EventEnvelope, NotUsed] =
```
Comment (johanandren) [inline core/src/main/scala/akka/persistence/couchbase/scaladsl/CouchbaseReadJournal.scala]: Scalafmt did it

File: core/src/main/scala/akka/persistence/couchbase/scaladsl/CouchbaseReadJournal.scala [hunk 2]
```diff
@@ -111,7 +111,7 @@ final class CouchbaseReadJournal(eas: ExtendedActorSystem, config: Config, confi
     } yield {
 
       val indexNames = indexes.map(_.name()).toSet
-      Set("tags-ordering-cover-meta").foreach(
+      Set("tags", "tag-seq-nrs").foreach(
```
Comment (johanandren) [inline core/src/main/scala/akka/persistence/couchbase/scaladsl/CouchbaseReadJournal.scala]: this is the actual fix, we had some old index name here.

**Comment by johanandren** [comment]
This is the thing I was talking about @renatocaval

File: core/src/main/scala/akka/persistence/couchbase/scaladsl/CouchbaseReadJournal.scala (hunk 2 above)
Comment (chbatey) [reply core/src/main/scala/akka/persistence/couchbase/scaladsl/CouchbaseReadJournal.scala]: Doh

File: core/src/main/scala/akka/persistence/couchbase/scaladsl/CouchbaseReadJournal.scala (hunk 1 above)
Comment (chbatey) [reply core/src/main/scala/akka/persistence/couchbase/scaladsl/CouchbaseReadJournal.scala]: It wasn't me

**Pull request merged**